Source/FakeItEasy.Net35/Compatibility.cs

```diff
@@ -51,28 +51,7 @@ where entry.Item1.IsAlive
     }
 }
 
-namespace System.ComponentModel.Composition
-{
-    using System.Diagnostics.CodeAnalysis;
-
-    [SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes", Justification = "Mimicks net40 BCL type.")]
-    [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass", Justification = "Tidier.")]
-    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface, AllowMultiple = true, Inherited = true)]
-    public class InheritedExportAttribute
-        : Attribute
-    {
-    }
-
-    [SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes", Justification = "Mimicks net40 BCL type.")]
-    [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass", Justification = "Tidier.")]
-    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
-    public class ImportManyAttribute
-        : Attribute
-    {
-    }
-}

```

Comment:
Not necessary to remove these in this step, but they are not used. A real PR would do this separately.